.github/workflows/deploy.yaml

```diff
@@ -204,8 +229,9 @@ jobs:
 
       - name: pip_install_mda
         run: |
-          pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple MDAnalysis
-          pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple MDAnalysisTests
+          ver=$(grep "__version__ =" package/MDAnalysis/version.py | awk '{print $3}')
```

Comment:
If you're using awk (instead of `cut`) then you can do it in one command
```bash
awk '/__version++ =/ {print $3; quit}' package/MDAnalysis/version.py
```
(I *think* the `quit` works.)

Reply:
done